tee-worker/enclave-runtime/src/rpc/worker_api_direct.rs

```diff
@@ -103,6 +106,32 @@ where
 		Ok(json!(json_value.to_hex()))
 	});
 
+	// author_getEnclaveId
+	let rsa_pubkey_name: &str = "author_getEnclaveId";
```

Comment:
I think `EnclaveId` is really confusing - how about `author_getEnclaveSignerAccount`